CHANGELOG.md

```diff
@@ -3,7 +3,17 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 2.0.2 - [XXXX-XX-XX]
+## X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+### `Fixed`
+
+### `Parameters`
+
+### `Changed`
+
```

Comment:
Not sure it actually matters, but maybe this part should be added _after_ the release is done.

Reply:
That's true, I will remove it and read it later